x-pack/platform/test/alerting_api_integration/security_and_spaces/group3/tests/maintenance_window/external/delete_maintenance_window.ts

```diff
@@ -76,6 +76,52 @@ export default function deleteMaintenanceWindowTests({ getService }: FtrProvider
               throw new Error(`Scenario untested: ${JSON.stringify(scenario)}`);
           }
         });
+
+        it('should handle delete disabled maintenance window request appropriately', async () => {
```

Comment:
I think we do not need this test at all. The delete API does not return an MW to check its status.